stracker/stracker/settings.py

```diff
@@ -103,9 +108,21 @@
 # Internationalization
 # https://docs.djangoproject.com/en/3.2/topics/i18n/
 
-LANGUAGE_CODE = 'en-us'
+LANGUAGE_CODE = "en"
+
+
+def gettext_noop(s):
+    return s
+
+
+LANGUAGES = [
+    ("en", gettext_noop("English")),
+    ("tr", gettext_noop("Turkish")),
+    ("de", gettext_noop("German")),
+    ("fr", gettext_noop("French")),
+]
```

Comment:
I think we should start a poll about which languages will we add. 
 
 In any case 
 LOCALE_PATHS = (str(BASE_DIR / "locale/"),)
 should be added.

Reply:
I think it should be as little as possible. It can be TR and EN.